_layouts/layout.htm

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html class="h-100" lang="en">
-	{% include _head.htm %}
-	<body class="h-100 d-flex flex-column">
-	{% include _header.htm %}
-	{{ content }}
-	{% include _footer.htm %}
+    {% include _head.htm %}
+    <body class="h-100">
+    {% include _header.htm %}
+    <div class="min-h-80">{{ content }}</div>
+    {% include _footer.htm %}
 
 	<!--   Core JS Files   -->
 	<script src="/assets/js/core/jquery-3.4.1.min.js" type="text/javascript"></script>
```

Comment:
Indentation changes make the Core JS Files Part not line up with the rest of the HTML

Reply:
Pushed a fix for the indentation.

Reply:
looks good